regolith/helpers/u_logurlhelper.py

```diff
@@ -52,16 +53,39 @@ def construct_global_ctx(self):
 
     def db_updater(self):
         rc = self.rc
-        key = f"{rc.name}"
+        key = f"{rc.projectum_id}"
         filterid = {'_id' : key}
-        projectum = rc.client.find_one(rc.database, rc.coll, filterid)
+        found_projectum = rc.client.find_one(rc.database, rc.coll, filterid)
+        # find all similar projectum ids
+        ids = []
+        for projectum in self.gtx["projecta"]:
+            if key in projectum.get("_id"):
+                ids.append(projectum.get("_id"))
 
-        if projectum is None:
-            raise RuntimeError(
-                "There does not seem to be a projectum with this name in this database"
-            )
+        if found_projectum is None and len(ids) == 0:
+            raise RuntimeError("Please input a valid projectum id or a valid fragment of a projectum id")
+        elif found_projectum is None and not rc.number:
+            print("There does not seem to be a projectum with this name in this database.")
+            print("However, there are projectums with similar names: ")
+            for i in range(len(ids)):
+                print(str(i+1) + ". " + str(ids[i]))
+            print("Please rerun the u_logurl helper with the same name as previously inputted, "
+                  "but with the addition of -n followed by a number corresponding to one of the above listed "
+                  "projectum ids that you would like to update.")
+        elif found_projectum is None:
+            if int(rc.number) < 1 or int(rc.number) > len(ids):
+                print("Sorry, you picked an invalid number")
```

Comment:
This is a bit elaborate.  I think it is probably better to throw an exception with a friendly error message rather do all this manual flow handling.